CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ The format is based on [Keep a Changelog], and this project adheres to [Semantic
 [Keep a Changelog]: https://keepachangelog.com/en/1.1.0/
 [Semantic Versioning]: https://semver.org/spec/v2.0.0.html
 
-## Unreleased
+<!-- TODO: Fix date -->
 
-**All Changes**: [`main`](https://github.com/TheBevyFlock/bevy_cli/commits/main)
+## v0.1.0-alpha - 2025-05-DD
```

Comment:
Hmm `0.1.0-alpha` creates the question "what do we call the next alpha release". From my perspective, all alphas should be leading up to an official 0.1.0 release (including a crates.io release), so it seems like the versioning should be `0.1.0-alpha.1`, `0.1.0-alpha.2`, etc
In public communications, we could consider shortening those to "Bevy CLI Alpha 1", `Bevy CLI Alpha 2", etc

Reply:
Yup I was thinking the same thing, I'm going to switch to v0.1.0-alpha.0 for this release.